tests/e2e/cluster_downgrade_test.go

```diff
@@ -76,6 +78,8 @@ func testDowngradeUpgrade(t *testing.T, clusterSize int) {
 		})
 	}
 	t.Logf("Cluster created")
+	generateSnapshot(t, 10, epc)
```

Comment:
Note, it might be useful to have both downgrade tests with and without snapshot. 
`TestDowngradeUpgradeClusterOf1` and `TestDowngradeUpgradeClusterOf1WithSnapshot`

Reply:
sg. updating.